src/bitwarden_exporter/bw_list_process.py

```
@@ -156,51 +119,55 @@ def process_list(settings: BitwardenExportSettings) -> None:
-    bw_current_status = json.loads(bw_exec(["status"], is_raw=False, settings=settings))
-    raw_items["status.json"] = bw_current_status
+    bw_current_status = json.loads(bw_exec(["status"], is_raw=False))
+    bw_process_items.raw_items.status.update(bw_current_status)
 
     if bw_current_status["status"] != "unlocked":
         raise BitwardenException("Vault is not unlocked")
     LOGGER.debug("Vault status: %s", json.dumps(bw_current_status))
 
-    bw_folders_dict = json.loads((bw_exec(["list", "folders"], is_raw=False, settings=settings)))
-    bw_folders: Dict[str, BwFolder] = {folder["id"]: BwFolder(**folder) for folder in bw_folders_dict}
+    bw_folders_dict = json.loads((bw_exec(["list", "folders"], is_raw=False)))
```

Ruff (0.14.2)

Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)